src/integrationTest/sam.test.ts

```diff
@@ -263,6 +264,13 @@ describe('SAM Integration Tests', async function() {
 
                 it('invokes and attaches on debug request (F5)', async function() {
                     setTestTimeout(this.test?.fullTitle(), 60000)
+                    // Allow previous sessions to go away.
+                    waitUntil(
+                        async function() {
```

Comment:
Nit: arrow function

Reply:
I think we should not spend time flip-flopping _anonymous_ arrow vs non-arrow functions. It deeply doesn't matter, except that arrow functions have a gotcha regarding `this`, which is the reason I default to non-arrow functions.

Reply:
We used to have linter rules enforcing arrow functions everywhere; the only time that we deemed non-arrow functions valid were for mocha tests, which is specifically recommended by mocha for access to mocha's `this`: https://mochajs.org/#arrow-functions ; it's why we use it five lines above.

We really should move all mocha tests to that, we were just too lazy about adding linter exceptions for each test.

Reply:
Why? The `this` gotcha is a disadvantage of arrow functions.